frontend/test/searchform.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+import SearchForm from '../app/components/searchform';
+import { AsyncTypeahead } from 'react-bootstrap-typeahead';
+
+describe('<SearchForm />', () => {
+  it('should render the typeahead search form', () => {
+/*
+    const props = {
+      options: [
+        { name: 'restaurant', camis: '123456', address: 'location'},
+        { name: 'restaurant2', camis: '223456', address: 'location2'}
+      ],
+      renderBusinessOptions: sinon.spy(),
+      handleSearch: sinon.spy()
+    }
+*/
+    const wrapper = shallow(<SearchForm />)
```

Comment:
FYI if you have components with proptypes set, all tests that don't set props within them and then pass them down to the enzyme wrapper ie `const props = blah blah blah` `const wrapper = shallow(<SearchForm {...props} />` will throw warnings when you run the test.